vernac/synterp.ml

```diff
@@ -293,8 +293,8 @@ let synterp_require from export qidl =
     let open Loadpath in
     match locate_qualified_library ?root qid with
     | Ok (dir,_) -> dir
-    | Error LibUnmappedDir -> raise (UnmappedLibrary (root, qid))
-    | Error LibNotFound -> raise (NotFoundLibrary (root, qid))
+    | Error LibUnmappedDir -> Loc.raise ?loc:qid.loc (UnmappedLibrary (root, qid))
+    | Error LibNotFound -> Loc.raise ?loc:qid.loc (NotFoundLibrary (root, qid))
```

Comment:
Actually the reason `Loc.raise` is not used here is that already the error datatype contains the loc, so special handling by upper layers is already possible.

But indeed being more precise here cannot hurt.

Reply:
I don't think that's true. `coqc` certainly doesn't try to do any special handling.

Reply:
What is not true?